server/server/middleware.py

```diff
@@ -13,12 +13,18 @@ def __call__(self, request):
         return self.get_response(request)
 
     def process_view(self, request, view_func, view_args, view_kwargs):
-        if getattr(view_func, "allow_unauthenticated", False):
+        third_party_allow_unauthenticated_url_names = [
+            "phac_aspc_helper_login"
```

Comment:
do we also need `phac_aspc_helper_authorize` ? I'm honestly not sure but we whitelisted both in cedars

Reply:
Ah, yup, looks like it would be necessary, good catch!